### PR TITLE
Refactor all buildImagePlaceholder_ to use createPlaceholderCallback

### LINKED ISSUE #3212: Refactor placeholder creation by using createPlaceholderCallback
Now that we have #2846 in place. All (except of `amp-youtube` due to a hack that can't be avoided currently) should start using a similar method of `amp-instagram` to create their dynamic placeholders using `createPlaceholderCallback`.

This includes `amp-brid-player`, `amp-jwplayer`, `amp-kaltura-player` and `amp-springboard-player`.

### PR DESCRIPTION
**Pull request by mkhatib**
(except YouTube for #3216)

Fixes #3212

### CODE REVIEW

File: extensions/amp-kaltura-player/0.1/amp-kaltura-player.js [hunk 1]
```diff
@@ -74,41 +76,24 @@ class AmpKaltura extends AMP.BaseElement {
...
-
-    this.applyFillContent(imgPlaceholder);
-    this.element.appendChild(imgPlaceholder);
-
-    this.loadPromise(imgPlaceholder).then(() => {
-      setStyles(imgPlaceholder, {
-        'visibility': '',
-      });
-    });
+    placeholder.setAttribute('noprerender', '');
```
Comment (erwinmombay) [inline extensions/amp-kaltura-player/0.1/amp-kaltura-player.js]: was this part of the eval that this shouldn't be prerendered?

Comment (mkhatib) [inline extensions/amp-kaltura-player/0.1/amp-kaltura-player.js]: I am actually not 100%. Sure was it only if the request is sending or setting cookies during prerender? I'll review the document. 

In this PR u added noprerender to items that uses the same domain for loading the main content and prerender content.

File: extensions/amp-brid-player/0.1/amp-brid-player.js [hunk 2]
```diff
@@ -112,34 +107,32 @@ class AmpBridPlayer extends AMP.BaseElement {
...
+    placeholderFallback.setAttribute('src',
+        'https://cdn.brid.tv/live/default/defaultSnapshot.png');
+    placeholderFallback.setAttribute('referrerpolicy', 'origin');
+    placeholderFallback.setAttribute('layout', 'fill');
+    placeholderFallback.setAttribute('fallback', '');
+    placeholder.appendChild(placeholderFallback);
+
+    // TODO(#5321): Investigate if this is actually the correct URL format for generated
+    // dynamic image.
+    placeholder.setAttribute('src',
```
Comment (mkhatib) [inline extensions/amp-brid-player/0.1/amp-brid-player.js]: Yes if alt is supported on the element we should propagate it. Will review to add it where supported

Comment (erwinmombay) [inline extensions/amp-brid-player/0.1/amp-brid-player.js]: thanks

File: extensions/amp-kaltura-player/0.1/amp-kaltura-player.js (hunk 1 above)
Comment (erwinmombay) [inline extensions/amp-kaltura-player/0.1/amp-kaltura-player.js]: SGTM

File: extensions/amp-brid-player/0.1/amp-brid-player.js (hunk 2 above)
Comment (mkhatib) [inline extensions/amp-brid-player/0.1/amp-brid-player.js]: We don't support `alt` attribute on most of these players.

File: extensions/amp-kaltura-player/0.1/amp-kaltura-player.js (hunk 1 above)
Comment (mkhatib) [inline extensions/amp-kaltura-player/0.1/amp-kaltura-player.js]: Checked the doc, doesn't seem we have problems with these, so dropping `noprerender` attribute from these.

**Comment by mkhatib** [comment]
PTAL 👀

**Comment by erwinmombay** [comment]
LGTM. @mkhatib thanks for double checking

**Pull request merged**